example/main.tf

```diff
@@ -3,7 +3,6 @@ provider "aws" {
   region              = "${var.aws_region}"
   access_key          = "${var.aws_access_key}"
```

Comment:
With the test fixture and example within the README, I think the example dir is redundant at this point. Going to delete it outright.